### PR TITLE
No more anti-tank racks/showcases.

### LINKED ISSUE #129: Anti tank  structures
All types of windows, display case, racks, jungle trufs, etc... everything except regular walls cant be destroyed by the tank when this pass over.

### PR DESCRIPTION
**Pull request by F-OS**
Racks/showcases no longer are invincible vs tanks.

closes #129

### CODE REVIEW

File: code/modules/vehicles/cm_armored.dm [hunk 1]
```diff
@@ -404,6 +404,14 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 		var/obj/structure/table/T = A
 		T.visible_message("<span class='danger'>[root] crushes [T]!</span>")
 		T.destroy(1)
+	else if(istype(A, /obj/structure/showcase))
+		var/obj/structure/showcase/S = A
+		S.visible_message("<span class='danger'>[root] bulldozes over [S]!</span>")
+		S.ex_act(1)
+	else if(istype(A, /obj/structure/rack))
+		var/obj/structure/window_frame/R = A
```
Comment (francinum) [inline code/modules/vehicles/cm_armored.dm]: var/obj/structure/rack/R

Comment (F-OS) [reply code/modules/vehicles/cm_armored.dm]: lol whoops

File: code/modules/vehicles/cm_armored.dm [hunk 2]
```diff
@@ -404,6 +404,14 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 		var/obj/structure/table/T = A
 		T.visible_message("<span class='danger'>[root] crushes [T]!</span>")
 		T.destroy(1)
+	else if(istype(A, /obj/structure/showcase))
+		var/obj/structure/showcase/S = A
+		S.visible_message("<span class='danger'>[root] bulldozes over [S]!</span>")
+		S.ex_act(1)
```
Comment (ninjanomnom) [inline code/modules/vehicles/cm_armored.dm]: `S.destroy(TRUE)`

File: code/modules/vehicles/cm_armored.dm [hunk 3]
```diff
@@ -404,6 +404,14 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 		var/obj/structure/table/T = A
 		T.visible_message("<span class='danger'>[root] crushes [T]!</span>")
 		T.destroy(1)
```
Comment (ninjanomnom) [inline code/modules/vehicles/cm_armored.dm]: Make this TRUE while you're here

File: code/modules/vehicles/cm_armored.dm [hunk 4]
```diff
@@ -404,6 +404,14 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
...
+	else if(istype(A, /obj/structure/showcase))
+		var/obj/structure/showcase/S = A
+		S.visible_message("<span class='danger'>[root] bulldozes over [S]!</span>")
+		S.ex_act(1)
+	else if(istype(A, /obj/structure/rack))
+		var/obj/structure/rack/R = A
+		R.visible_message("<span class='danger'>[root] smashes through the [R]!</span>")
+		R.destroy()
```
Comment (ninjanomnom) [inline code/modules/vehicles/cm_armored.dm]: `R.destroy(TRUE)`

**Pull request merged**